libqtile/widget/base.py

```diff
@@ -496,6 +496,12 @@ class _TextBox(_Widget):
             "Setting ``scroll_fixed_width=True`` will force the widget to have a fixed width, regardless of the size of the text.",
         ),
         ("rotate", True, "Rotate text in vertical bar."),
+        (
+            "vertical_text_direction", 
+            "default", 
```

Comment:
```suggestion
            "vertical_text_direction",
            "default",
```
Need to remove trailing spaces.